lib/modules/manager/kustomize/extract.ts

```diff
@@ -133,7 +133,12 @@ export function parseKustomize(content: string): Kustomize | null {
     return null;
   }
 
-  if (!['Kustomization', 'Component'].includes(pkg.kind)) {
+  const pkgKindKustomization = 'Kustomization';
+  const pkgKindComponent = 'Component';
+
+  pkg.kind = pkg.kind || pkgKindKustomization;
+
+  if (![pkgKindKustomization, pkgKindComponent].includes(pkg.kind)) {
```

Comment:
```suggestion
  if (is.string(pkg.kind) && !['Kustomization', 'Component'].includes(pkg.kind)) {
```

Simplify

Reply:
I guess this would break the fallback logic right? We want the correct `depType`.

Reply:
@viceice OK, done!